GLTFSDK/Inc/GLTFSDK/Schema.h

```diff
@@ -11,8 +11,39 @@ namespace Microsoft
     {
-        //TODO: add these variables for all the schema uris (and use in the schemaFlagsMap)?
         constexpr const char SCHEMA_URI_GLTF[] = "glTF.schema.json";
+        constexpr const char SCHEMA_URI_GLTFID[] = "glTFid.schema.json";
+        constexpr const char SCHEMA_URI_GLTFCHILDOFROOTPROPERTY[] = "glTFChildOfRootProperty.schema.json";
+        constexpr const char SCHEMA_URI_GLTFPROPERTY[] = "glTFProperty.schema.json";
+        constexpr const char SCHEMA_URI_BUFFER[] = "buffer.schema.json";
+        constexpr const char SCHEMA_URI_BUFFERVIEW[] = "bufferView.schema.json";
+        constexpr const char SCHEMA_URI_ACCESSOR[] = "accessor.schema.json";
+        constexpr const char SCHEMA_URI_ACCESSORSPARSE[] = "accessor.sparse.schema.json";
+        constexpr const char SCHEMA_URI_ACCESSORSPARSEVALUES[] = "accessor.sparse.values.schema.json";
+        constexpr const char SCHEMA_URI_ACCESSORSPARSEINDICES[] = "accessor.sparse.indices.schema.json";
+        constexpr const char SCHEMA_URI_ASSET[] = "asset.schema.json";
+        constexpr const char SCHEMA_URI_SCENE[] = "scene.schema.json";
+        constexpr const char SCHEMA_URI_NODE[] = "node.schema.json";
+        constexpr const char SCHEMA_URI_MESH[] = "mesh.schema.json";
+        constexpr const char SCHEMA_URI_MESHPRIMITIVE[] = "mesh.primitive.schema.json";
+        constexpr const char SCHEMA_URI_SKIN[] = "skin.schema.json";
+        constexpr const char SCHEMA_URI_CAMERA[] = "camera.schema.json";
+        constexpr const char SCHEMA_URI_CAMERAORTHOGRAPHIC[] = "camera.orthographic.schema.json";
+        constexpr const char SCHEMA_URI_CAMERAPERSPECTIVE[] = "camera.perspective.schema.json";
+        constexpr const char SCHEMA_URI_MATERIAL[] = "material.schema.json";
+        constexpr const char SCHEMA_URI_MATERIALNORMALTEXTUREINFO[] = "material.normalTextureInfo.schema.json";
+        constexpr const char SCHEMA_URI_MATERIALOCCLUSIONTEXTUREINFO[] = "material.occlusionTextureInfo.schema.json";
+        constexpr const char SCHEMA_URI_MATERIALPBRMETALLICROUGHNESS[] = "material.pbrMetallicRoughness.schema.json";
+        constexpr const char SCHEMA_URI_TEXTURE[] = "texture.schema.json";
+        constexpr const char SCHEMA_URI_TEXTUREINFO[] = "textureInfo.schema.json";
+        constexpr const char SCHEMA_URI_IMAGE[] = "image.schema.json";
+        constexpr const char SCHEMA_URI_SAMPLER[] = "sampler.schema.json";
+        constexpr const char SCHEMA_URI_ANIMATION[] = "animation.schema.json";
+        constexpr const char SCHEMA_URI_ANIMATIONSAMPLER[] = "animation.sampler.schema.json";
+        constexpr const char SCHEMA_URI_ANIMATIONCHANNEL[] = "animation.channel.schema.json";
+        constexpr const char SCHEMA_URI_ANIMATIONCHANNELTARGET[] = "animation.channel.target.schema.json";
+        constexpr const char SCHEMA_URI_EXTENSION[] = "extension.schema.json";
+        constexpr const char SCHEMA_URI_EXTRAS[] = "extras.schema.json";
 
         enum class SchemaFlags : uint64_t
```

Comment:
This definitely feels redundant now that you have added all these:
`constexpr const char SCHEMA_URI_GLTFID[] = "glTFid.schema.json";`